samples: matter: Enabled BLE extended announcement (#15371)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

To enable the Wi-Fi power save mode, set the :kconfig:option:`CONFIG_NRF_WIFI_LOW_POWER` Kconfig option to ``y``.

Configure Bluetooth LE advertising duration
*******************************************

The Matter uses Bluetooth® Low Energy (LE) to advertise its service for the device commissioning purposes.
The duration of this advertising is configurable and can last up to 15 minutes in the standard mode and up to 48 hours in the Extended Announcement mode.
Extending the advertising duration may improve the user experience, as it gives more time for the user to setup the device, but it also increases the energy consumption.

Selecting the optimal advertising duration is a compromise and depends on the specific application use case.

> To reduce the consumed energy you can consider:

Author: I added to your suggestion "and reduce the consumed energy", because I think it's important to highlight what is the purpose of the configuration.

> * Setting :kconfig:option:`CONFIG_CHIP_BLE_EXT_ADVERTISING` to ``n`` to disable Extended Announcement (also called Extended Beaconing), which allows a device to advertise for a duration of more than 15 minutes.

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_CHIP_BLE_EXT_ADVERTISING` -  Set to ``n`` to disable Extended Announcement (also called Extended Beaconing).
  When disabled, the device is not allowed to advertise for a duration of more than 15 minutes.
```

> * Setting :kconfig:option:`CONFIG_CHIP_BLE_ADVERTISING_DURATION` to the lower value.

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_CHIP_BLE_ADVERTISING_DURATION` - Set how long the device advertises (in minutes).
```
